### ISSUE TITLE
top menu buttons don't work

### ISSUE THREAD

**Issue by SquareCereal724** [FriendUPCloud/friendup#178]
![image](https://github.com/FriendUPCloud/friendup/assets/83764123/d08a1b10-1331-4bc5-be20-f9783bb4f9a1)

**Comment by SquareCereal724** [comment]
none of the buttons do anything in system, file, window and help

**Comment by titlestad** [comment]
Is this on a touch screen laptop? This is one I really want to fix, but I need to acquire some hybrid touch laptop to test. Or does anyone know of any way to test this without having the hardware?

**Comment by SquareCereal724** [comment]
yes my laptop is a touchscreen. I'll try to let you know if i find a solution.
